app/src/common/shared/com/igalia/wolvic/ui/widgets/HorizontalTabsBar.java

```diff
@@ -0,0 +1,76 @@
+package com.igalia.wolvic.ui.widgets;
+
+import android.content.Context;
+import android.widget.Button;
+
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.igalia.wolvic.R;
+import com.igalia.wolvic.browser.SessionChangeListener;
+import com.igalia.wolvic.browser.SettingsStore;
+import com.igalia.wolvic.browser.engine.SessionStore;
+import com.igalia.wolvic.ui.adapters.TabsBarAdapter;
+
+public class HorizontalTabsBar extends AbstractTabsBar implements SessionChangeListener {
```

Comment:
I don't think the implements is needed, isn't it?

Reply:
Good catch, the superclass already implements it.